install.sh

```diff
@@ -143,6 +148,7 @@ User=
 Group=
 WorkingDirectory=$rprefix
 EOS
+    chmod 644 "$rsystemd"/scylla-jmx.service.d/nonroot.conf
```

Comment:
This is slightly wrong. If we're not root, the user is installing into their own directory, and we don't need group permissions. If umask removes group permissions then they probably want it that way.

However, fixing this is way more trouble than it's worth. Probably no one installs without root these days, it's all docker or package install. So I'll merge it as is.